test/slow/test-invalid-api-key.js

```diff
@@ -1,5 +1,5 @@
 var common = require('../common');
-var airbrake = require(common.dir.root).createClient('invalidkey', 'production');
+var airbrake = require(common.dir.root).createClient('1234', 'invalid', 'production');
 var assert = require('assert');
 
 var myErr = new Error('test-notify');
```

Comment:
Commenting on line 8 as our thread disappeared.

> @shime 
> Thanks, updated.

Wait what? How is it possible that this assertion passes both with `400` and `401`?